tools/integration_tests/util/operations/dir_operations.go

```diff
@@ -160,23 +169,23 @@ func DirSizeMiB(dirPath string) (dirSizeMB int64, err error) {
 	return dirSizeMB, err
 }
 
-func DeleteManagedFoldersInBucket(managedFolderPath, bucket string, t *testing.T) {
+func DeleteManagedFoldersInBucket(managedFolderPath, bucket string) {
 	gcloudDeleteManagedFolderCmd := fmt.Sprintf("alpha storage rm -r gs://%s/%s", bucket, managedFolderPath)
 
 	_, err := ExecuteGcloudCommandf(gcloudDeleteManagedFolderCmd)
 	if err != nil && !strings.Contains(err.Error(), "The following URLs matched no objects or files") {
-		t.Fatalf(fmt.Sprintf("Error while deleting managed folder: %v", err))
+		log.Fatalf(fmt.Sprintf("Error while deleting managed folder: %v", err))
 	}
 }
 
-func CreateManagedFoldersInBucket(managedFolderPath, bucket string, t *testing.T) {
+func CreateManagedFoldersInBucket(managedFolderPath, bucket string) {
 	// Delete if already exist.
-	DeleteManagedFoldersInBucket(managedFolderPath, bucket, t)
+	DeleteManagedFoldersInBucket(managedFolderPath, bucket)
 	gcloudCreateManagedFolderCmd := fmt.Sprintf("alpha storage managed-folders create gs://%s/%s", bucket, managedFolderPath)
 
 	_, err := ExecuteGcloudCommandf(gcloudCreateManagedFolderCmd)
 	if err != nil {
-		t.Fatalf(fmt.Sprintf("Error while creating managed folder: %v", err))
+		log.Fatalf(fmt.Sprintf("Error while creating managed folder: %v", err))
```

Comment:
Why did we move from t.Fatalf to log.FatalF?

Reply:
Because we wanted to make it independent of t *testing.T, We required to call this function out of tests.